app/main.py

```diff
@@ -1 +1,52 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self, name: str, appetite: int, is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            if self.appetite != 0:
```

Comment:
Use `and` instead of two nested ifs. it makes code more readable
DO NOT:
```
if condition1:
    if condition2:
         code ... 
```
DO:
```
if condition1 and consition2:
    code ....
```